.github/workflows/cmake.yml

```diff
@@ -87,3 +87,33 @@ jobs:
         cmake -B BUILD/RELEASE/BUILD .
         cmake --build BUILD/RELEASE/BUILD --config RELEASE --parallel
         cmake --install BUILD/RELEASE/BUILD --config RELEASE
+  msys2-ucrt64:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+      - uses: actions/checkout@v5
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: UCRT64
+          update: true
+          install: git mingw-w64-ucrt-x86_64-gcc mingw-w64-ucrt-x86_64-cmake mingw-w64-ucrt-x86_64-ninja
+      - name: CI-Build
+        run: |
+          ./msys2-ci-build.sh
+  msys2-clangarm64:
+    runs-on: windows-11-arm
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+      - uses: actions/checkout@v5
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: CLANGARM64
+          update: true
+          install: git mingw-w64-clang-aarch64-clang  mingw-w64-clang-aarch64-cmake mingw-w64-clang-aarch64-ninja
+      - name: CI-Build
+        run: |
+          ./msys2-ci-build.sh
```

Comment:
I don't think this script should be at the top level.

Maybe we should consider moving the ```docker/``` folder into a ```ci/``` folder and put everything there or introduce a ```utils/``` folder where this kind of script can go.